app/src/main/java/com/zulip/android/activities/RecyclerMessageAdapter.java

```diff
@@ -289,7 +289,7 @@ public boolean addOldMessage(Message message, int messageAndHeadersCount, String
      */
     public void addNewMessage(Message message) {
         MessageHeaderParent item = null;
-        for (int i = getItemCount(false) - 1; i > 1; i--) {
+        for (int i = getItemCount(false) - 1; i >= 1; i--) {
```

Comment:
you're correct; when I tried the changes, new messages were not being loaded (might be related to #226) 

double checked the logic; works well 👍